src/monocle_apptrace/wrap_common.py

```diff
@@ -75,7 +76,17 @@ def get_embedding_model_for_vectorstore(instance):
         'type': VECTOR_STORE,
     },
 }
+def get_embedding_model_haystack(instance):
```

Comment:
Can this be moved to metamodel?

Reply:
embedding model in coming in pipeline package and we need it in retriever package , thats why we cannot remove this code